example/apisix/plugins/prometheus/exporter.lua

```diff
@@ -0,0 +1,34 @@
+--
```

Comment:
As this file is test-only, we can move it under t/lib?

Reply:
done